auraed/src/cells/cell_service/cells/cgroups/cpuset/cpus.rs

```diff
@@ -76,12 +67,9 @@ mod tests {
     #[test]
     fn test_validation_success(input: &str) {
         let input: Cpus = Cpus::new(input.into());
```

Comment:
I know this is from before, but it seems weird that we use `Cpus::new` and then call `.into_inner()` on ln 71. Why don't we just call `input.to_string()` on 71?

We do this in other tests as well.